plutarch-test/plutarch-test.cabal

```diff
@@ -94,17 +95,27 @@ common deps
 
 library
   import:          c, deps
-  hs-source-dirs:  src
+  hs-source-dirs:  plutarch-base
   exposed-modules: Plutarch.Test
   other-modules:
     Plutarch.Test.Benchmark
     Plutarch.Test.Golden
     Plutarch.Test.ListSyntax
 
-executable plutarch-test
+executable plutarch-extra
```

Comment:
I'd call it `plutarch-extra-test` to be clear that the executable will be running tests.